plugin/vim-erlang-tags.vim

```diff
@@ -54,13 +54,19 @@ function! VimErlangTagsSelect(split)
     if a:split
         split
     endif
-    let orig_isk = &isk
+    let a:orig_isk = &isk
```

Comment:
Why the "a:" prefix? That is supposed to be used for accessing arguments (see :help a:var).